src/components/Project.vue

```diff
@@ -1,187 +1,188 @@
-                <b-input-group-append>
-                <b-button variant="success" @click="backProject">Back!</b-button>
-                </b-input-group-append>
-            </b-input-group>
-        </b-modal>
-    </div>
+    <b-modal title="Back this project" id="backs-modal" hide-footer>
+      <b-input-group prepend="$" class="mt-3">
+        <b-form-input v-model="backs_amount"></b-form-input>
+        <b-input-group-append>
+          <b-button variant="success" @click="backProject">Back!</b-button>
+        </b-input-group-append>
+      </b-input-group>
+    </b-modal>
+  </div>
 </template>
 
 <script>
-    import axios from "axios";
-
-    export default {
-        name: "Project",
-        data() {
-            return {
-                project: null,
-                rewards: null,
-                backDialogVisible: false,
-                backs_amount: 0,
-            }
-        },
-        beforeMount() {
-            this.loadProject()
-            this.loadRewards()
-        },
-        methods: {
-            loadProject() {
-                axios
-                    .get("http://localhost:3000/project/" + this.$route.params.projectName)
-                    .then((response) => {
-                        // console.log(response.data);
-                        this.project = response.data
-                    })
-                    .catch((error) => {
-                        // Failure
-                        alert(error);
-                    });
-            },
-            loadRewards() {
-                axios
-                    .get("http://localhost:3000/project/" + this.$route.params.projectName + "/rewards")
-                    .then((response) => {
-                        // console.log(response.data);
-                        this.rewards = response.data
-                    })
-                    .catch(error => {
-                        alert(error)
-                    })
-            },
-            dayTillDeadline() {
-                const date1 = new Date();
-                const date2 = new Date(this.project.project_deadline);
-                const diffTime = Math.abs(date2 - date1);
-                const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
-                return diffDays;
-            },
-            getFormattedDate(date) {
-                    var year = date.getFullYear();
+import axios from "axios";
 
-                    var month = (1 + date.getMonth()).toString();
-                    month = month.length > 1 ? month : '0' + month;
+export default {
+  name: "Project",
+  data() {
+    return {
+      project: null,
+      rewards: null,
+      backDialogVisible: false,
+      backs_amount: 0
+    };
+  },
+  beforeMount() {
+    this.loadProject();
+    this.loadRewards();
+  },
+  methods: {
+    loadProject() {
+      axios
+        .get("http://localhost:3000/project/" + this.$route.params.projectName)
```

Comment:
For the project name, I understand that it was @rafflesandrison's idea to use `this.$route.params.projectName` but I was wondering whether we could store it in `data` as a variable instead.